docs/running-aggregate-with-docker.md

```diff
@@ -0,0 +1,57 @@
+# Running Aggregate with Docker
+
+## Requirememts
+
+- Install [Docker](https://www.docker.com)
+
+## Quick start - External PostgreSQL instance
+
+- Build the Docker image with `./gradlew clean dockerBuild -xtest -pwarMode=complete`.
+
+  Take note of the tag of the created image. It will appear in a message like this one:
+  
+  ```
+  ...
+  Successfully built bf09fb92a6e9
+  Successfully tagged aggregate:v1.6.0-beta.0-dirty
+  ```
+
+- Launch Aggregate for the first time with `docker run -d -p 8080:8080 -it --name=aggregate aggregate:v1.6.0-beta.0-dirty`.
```

Comment:
This doesn't seem to work for me. I get the previous message...
```
Successfully built 172dc9613b65
Successfully tagged aggregate:v1.5.0-24-g8bce4afd
```

```
$ docker run -d -p 8080:8080 -it --name=aggregate aggregate:v1.5.0-24-g8bce4afd
7f700ba4b1b1c4e04fe71fd281d1a366cb768ddcdbd0dbaaec7709b4cb4fd2f6
docker: Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"docker-entrypoint.sh\": executable file not found in $PATH": un
```

Reply:
That is really strange, because the `Dockerfile` places that file in `/usr/local/bin`, which is in the $PATH.

Could you run `docker -it --entrypoint /bin/bash aggregate:v1.5.0-24-g8bce4afd`? It will start a shell inside the container. Then run `echo $PATH` and copy here the output, please?

Reply:
```
$ docker run -it --entrypoint /bin/bash aggregate:v1.5.0-24-g8bce4afd
bash-4.4# echo $PATH
/usr/local/tomcat/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/lib/jvm/java-1.8-openjdk/jre/bin:/usr/lib/jvm/java-1.8-openjdk/bin

Reply:
I can also confirm that the file is there

```
bash-4.4# ls -lah /usr/local/bin/docker-entrypoint.sh
-rw-r--r--    1 root     root        2.2K Jul  2 22:49 /usr/local/bin/docker-entrypoint.sh
```

Reply:
Thanks!